### PR TITLE
[no ticket][risk=no] Puppeteer Delete workspace

### PR DESCRIPTION
**Pull request by aweng98**
Clean up test generated data: Delete cloned workspaces in `workspace-clone.spec.ts` test.
`workspace-delete.spec.ts` is obsolete.

### CODE REVIEW

File: e2e/app/page/workspace-data-page.ts [hunk 1]
```diff
@@ -45,11 +44,6 @@ export default class WorkspaceDataPage extends WorkspaceBase {
     ]);
   }
 
-  async selectWorkspaceAction(action: EllipsisMenuAction): Promise<void> {
```
Comment (aweng98) [inline e2e/app/page/workspace-data-page.ts:48]: `selectWorkspaceAction` moved to `workspace-base`.

File: e2e/app/page/workspaces-page.ts [hunk 2]
```diff
@@ -157,18 +156,6 @@ export default class WorkspacesPage extends WorkspaceEditPage {
     return notebookPage.goAnalysisPage();
   }
 
-  /**
-   * Click DELETE WORKSPACE button in delete workspace Confirmation dialog.
-   * @return {string} Dialog textContent.
-   */
-  async deleteWorkspaceModal(): Promise<string[]> {
```
Comment (aweng98) [inline e2e/app/page/workspaces-page.ts:164]: `deleteWorkspaceModal` moved to `workspace-base` and renamed to `dismissDeleteWorkspaceModal`.

File: e2e/app/page/workspace-base.ts [hunk 3]
```diff
@@ -212,5 +213,37 @@ export default abstract class WorkspaceBase extends AuthenticatedPage {
...
+  /**
+   * Dismss Delete Workspace Confirmation modal by click a button.
+   * @return {LinkText} clickButtonText Button to click.
+   */
+  async dismissDeleteWorkspaceModal(clickButtonText: LinkText = LinkText.DeleteWorkspace): Promise<string[]> {
+    const modal = new Modal(this.page);
+    const contentText = await modal.getTextContent();
+    await modal.clickButton(clickButtonText, {waitForClose: true});
+    await waitWhileLoading(this.page);
+    console.log(`Dismiss Delete Workspace modal by click ${clickButtonText} button`);
```
Comment (gjuggler) [inline e2e/app/page/workspace-base.ts:245]: nit: remove console.log before merging

**Review by gjuggler — APPROVED** [review]

File: e2e/app/page/workspace-base.ts (hunk 3 above)
Comment (aweng98) [reply e2e/app/page/workspace-base.ts]: Done

**Review by aweng98 — COMMENTED** [review]
Thank you

**Pull request merged**